### PR TITLE
Add ENS support for sepolia

### PR DESCRIPTION
**Pull request by ChaituVR**
Related to https://github.com/snapshot-labs/snapshot/issues/4614

## How to test:
- Change .env to 
```
VITE_ENV=demo
VITE_DEFAULT_NETWORK=11155111
```
- When you go to http://localhost:8080/#/setup?step=1 you should see ENS names from sepolia
- When you go to your space, you should be able to get controller from sepolia 
- You should be able to create new spaces using sepolia ENS

### CODE REVIEW

File: src/components/SetupDomain.vue [hunk 1]
```diff
@@ -126,7 +126,7 @@ onUnmounted(() => clearInterval(waitingForRegistrationInterval));
       />
 
       <BaseMessageBlock
-        v-if="defaultNetwork === '5'"
+        v-if="defaultNetwork && env === 'demo'"
```
Comment (samuveth) [inline src/components/SetupDomain.vue:129]:
```suggestion
        v-if="env === 'demo'"
```
No point in checking for defaultNetwork

Comment (samuveth) [inline src/components/SetupDomain.vue:129]:
```suggestion
        v-if="env === 'demo'"
```
Do you need this check?

**Comment by samuveth** [comment]
@ChaituVR when are we planning to merge this?

**Comment by ChaituVR** [comment]
@samuveth can merge it right now if approved, can update .env on the date we announce

**Review by bonustrack — APPROVED** [review]
utACK

**Pull request merged**